ovos_persona_server/persona.py

```
@@ -0,0 +1,68 @@
+    Raises:
+        HTTPException: If persona configuration or loading fails.
+    """
+    global default_persona
+    if default_persona is None:
+        persona_data = settings.persona_config
+        print(persona_data)
+        try:
+            default_persona = Persona(persona_data["name"], persona_data)
+        except json.JSONDecodeError:
+            raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                                detail=f"Invalid JSON config for persona: {persona_data}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)